### PR TITLE
Merge 3.x

### PR DESCRIPTION
**Pull request by greg0ire**
Maintenance merge

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -33,29 +34,33 @@ env:
...
+    - php: '7.1'
       env: TARGET=docs
-    - php: '7.0'
+    - php: '7.1'
       env: TARGET=lint
     - php: '5.6'
       env: COMPOSER_FLAGS="--prefer-lowest"
-    - php: '7.0'
+    - php: '7.1'
+      env: SYMFONY=2.3.*
```
Comment (jordisala1991) [inline .travis.yml]: This should be removed on merge

Comment (greg0ire) [inline .travis.yml]: Lol I just noticed it

Comment (greg0ire) [inline .travis.yml]: While doing a similar merge on news bundle

File: Formatter/Pool.php [hunk 2]
```diff
@@ -33,7 +33,7 @@ class Pool implements LoggerAwareInterface
     protected $defaultFormatter;
 
     /**
-     * @param string $defaultFormatter
+     * @param LoggerInterface will be used to report errors
```
Comment (jordisala1991) [inline Formatter/Pool.php]: this one is strange. This is not a loggerInterface or atleast not by its name: `defaultFormatter`

Comment (greg0ire) [inline Formatter/Pool.php]: fixed

**Comment by OskarStark** [comment]
Thanks @greg0ire 👍

**Pull request merged**